### PR TITLE
Add play.abbelsmp.de

### PR DESCRIPTION
**Pull request by Vinnipapi**
## Type of change

- [✔️ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [✔️ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [✔️ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [✔️] I have created the manifest.json with the required values

## Further comments
It is just an survial server based on the standard minecraft features with some plugins, no lobby, just a small spawn island.

### CODE REVIEW

File: minecraft_servers/abbelsmp/background.png
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/background.png:1]: Compress

File: minecraft_servers/abbelsmp/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/background@2x.png:1]: Compress

File: minecraft_servers/abbelsmp/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "abbelsmp",
+    "nice_name": "AbbelSMP",
+    "direct_ip": "play.abbelsmp.de",
+    "server_wildcards": [
+      "%.abbelsmp.de"
```
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/manifest.json:6]: Not a real wildcard
https://i.imgur.com/wIwIPum.png

File: minecraft_servers/abbelsmp/manifest.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "nice_name": "AbbelSMP",
+    "direct_ip": "play.abbelsmp.de",
+    "server_wildcards": [
+      "%.abbelsmp.de"
+    ],
+    "supported_languages": [
+       "de"
+    ],
+    "social": {
+      "twitter": "vinnipapi",
```
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/manifest.json:12]: Links to private twitter account; either server account or remove

File: minecraft_servers/abbelsmp/manifest.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      "tiktok": "vinnipapii",
```
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/manifest.json:13]: Links to private tiktok account; either server account or remove

File: minecraft_servers/abbelsmp/manifest.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      "instagram": "vinnipapi",
```
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/manifest.json:14]: Links to private instagram account; either server account or remove

File: minecraft_servers/abbelsmp/manifest.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      "discord": "https://discord.gg/qXPaAGs8ru",
+      "youtube": "https://www.youtube.com/channel/UCSamgE1KYvC7qZn56T0J2yg"
```
Comment (DoJapHD) [inline minecraft_servers/abbelsmp/manifest.json:16]: That's our YouTube channel???

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/abbelsmp/background.png
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/background.png:1]: Done

File: minecraft_servers/abbelsmp/background@2x.png
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/background@2x.png:1]: Done

File: minecraft_servers/abbelsmp/manifest.json (hunk 1 above)
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/manifest.json]: Removed

File: minecraft_servers/abbelsmp/manifest.json (hunk 2 above)
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/manifest.json]: Removed

File: minecraft_servers/abbelsmp/manifest.json (hunk 3 above)
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/manifest.json]: Removed

File: minecraft_servers/abbelsmp/manifest.json (hunk 4 above)
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/manifest.json]: Removed

File: minecraft_servers/abbelsmp/manifest.json (hunk 5 above)
Comment (Vinnipapi) [reply minecraft_servers/abbelsmp/manifest.json]: Removed

**Review by Vinnipapi — COMMENTED** [review]
Changed everything

**Comment by DoJapHD** [comment]
You created a folder but it should be in the "minecraft_server" directory

**Comment by Vinnipapi** [comment]
Soryy I did not saw it. Know everything should be fine :D

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**